fetch_data.py

```diff
@@ -501,3 +557,49 @@ def cftime_value_to_ymd(time_value, base_date):
     """Convert a time value in days since the base date to a year, month, day tuple."""
     date = base_date + datetime.timedelta(days=time_value)
     return date.year, date.month, date.day
+
+
+def get_place_data(place_id):
```

Comment:
I think here we can lean on the `vectordata.py` route which looks like it already has this functionality baked in.  Could a simple function be added there, "get_community_by_id" that returns whatever is present in the `all_communities_full` variable that looks like it's populated at startup?